takwimu/static/css/takwimu.scss

```diff
@@ -255,17 +255,23 @@ a {
     position: initial;
   }
 
-  .dropdown-toggle::after{
-    font-family: 'Font Awesome 5 Free';
-    font-weight: 900;
-    content: '\f107';
-    border: 0;
-    vertical-align: 0;
-    visibility: hidden;
-    margin-left: 0;
-  }
-  .nav-link svg {
-    margin-left: .255em;
+  .dropdown-toggle {
+    white-space: nowrap;
```

Comment:
@DavidLemayian 
This seems to be a good approach on making sure the `navbar items` do no `wrap`, 
But am skeptical if it's a good fit, Should reduce the padding for each item in `navbar` or ? :thinking: